dctest-meows-runner/Dockerfile

```diff
@@ -47,6 +47,7 @@ RUN apt-get update \
             time \
             kmod \
             iptables \
+            iproute2 \
             openssh-client \
             sudo \
             hub \
```

Comment:
unrelated, but `hub` will not be used.

Reply:
Thank you.
I also tried to run the job of creating PR with neco-tenant-apps on this image and forgot to delete it.
Indeed, this is not used now, so I will delete it too.

Reply:
If we really want this, I'd suggest `gh` instead of `hub`.